docusaurus/docs/Android/02-client/06-guides/11-video-agora.mdx

```diff
@@ -0,0 +1,798 @@
+# Agora: Video Integration Guide for Android
+
+This page shows you how to integrate [Agora](https://www.agora.io/en/) into Stream Chat SDK for implementing video call features.
+
+## Introduction
+Video calls have become immensely popular since the onset of the pandemic. Today, we take a look at how you can use the service of agora to integrate video calls into the Stream Chat SDK.
```

Comment:
Maybe add a link for the word [agora] (Also maybe caps it)

Reply:
Nice catch! done: [fadcb87](https://github.com/GetStream/stream-chat-android/pull/4205/commits/fadcb87f8bcc2d89c69c03b595e230a4c44bc197)